src/Zip.Shared/Shared.cs

```diff
@@ -599,7 +634,7 @@ internal static int ReadWithRetry(System.IO.Stream s, byte[] buffer, int offset,
                     n = s.Read(buffer, offset, count);
                     done = true;
                 }
-#if NETCF || SILVERLIGHT
+#if NETCF || SILVERLIGHT || NETSTANDARD2_0 || NETCOREAPP2_0
```

Comment:
Do you happen to know why this is #if 'd out for .NET Framework? Curious why .NET Core needs this and .NET Framework doesn't.

Reply:
I was worried about the _HRForException method not working under .NET Standard, due to its usage of the Marshal class, but after some research, it seems I was a bit overzealous lol. I'll go back and take that out.